lib/winston-logstash-udp.js

```diff
@@ -70,11 +71,14 @@ LogstashUDP.prototype.log = function(level, msg, meta, callback) {
         json: true
     });
 
-    self.sendLog(logEntry, function() {
-        self.emit('logged', true);
-        callback(null, true);
-    });
+    // There is no need to wait for a callback as there is no guarantee that
+    // message will ever reach destination and we never get the feedback with UDP.
+    // Having self.emit('logged', true) as part of callback in sendLog would
+    // cause it to take 10% more time.
+    self.sendLog(logEntry);
```

Comment:
I Agree to having this optional. maybe even turned on to check for successful sending by default.